modules/oe_paragraphs_document/oe_paragraphs_document.info.yml

```diff
@@ -0,0 +1,18 @@
+name: OpenEuropa Paragraphs Document
+type: module
+description: Provides a paragraph type that references document media entities.
+package: OpenEuropa
+core_version_requirement: ^8.9 || ^9.1
+
+dependencies:
+  - oe_paragraphs:oe_paragraphs
```

Comment:
This dependency is not needed per se. It should be replaced by oe_media:oe_media as this submodule depends on the document media bundle provided there.

We can drop the oe_paragraphs dependency as there's nothing needed from the main module per se. We can add a direct dependency to paragraphs itself.